src/IdnaEncoder.php

```diff
@@ -92,11 +92,17 @@ public static function to_ascii($string) {
 		// Step 4: Check if it's ASCII now
 		if (self::is_ascii($string)) {
 			// Skip to step 7
+			/*
+			 * As the `nameprep()` method returns the original string, this code will never be reached until
+			 * that method is properly implemented.
+			 */
+			// @codeCoverageIgnoreStart
```

Comment:
I think this would be clearer if the entire construct is a single comment entity.
```suggestion
			/*
			 * As the `nameprep()` method returns the original string, this code will never be reached until
			 * that method is properly implemented.
			 *
			 * @codeCoverageIgnoreStart
			 */
```

Reply:
@schlessera I 100% agree. Unfortunately, that doesn't work.

![image](https://user-images.githubusercontent.com/663378/134523327-83934fde-8ec0-4b5a-8c22-95a9e0bd9127.png)

versus

![image](https://user-images.githubusercontent.com/663378/134523510-5693e02f-c424-464d-a3a6-c4e6c9bbc37b.png)